### PR TITLE
Reordered, improved and moved change log entries

### PR DESCRIPTION
**Pull request by andy-maier**
Ready for review and merge.

### CODE REVIEW

File: docs/changes.rst [hunk 1]
```diff
@@ -173,11 +169,26 @@ Incompatible changes
...
 
-* Fix issue where mof. compile_str could not compile mof that was
-  defined through a mof file containing  #pragma include statements.
-  This precluded using a string to define the classes to include in
-  a mof compile in a string and required that the include be a file.
-  See issue #1138
+* Revamped the (experimental) logger configuration mechanism completely.
+  It remains experimental. See issue #859. The changes include:
+
+  - Created 3 methods in `WBEMConnection` that allow pywbem logs to be
```
Comment (KSchopmeyer) [inline docs/changes.rst]: Publically we have one method

File: docs/changes.rst [hunk 2]
```diff
@@ -410,56 +392,41 @@ Enhancements
...
-  are not parts of instances.  Also changed code in IterQueryInstances were
-  parameters that are required by the called ExecQuery and OpenQueryInstances
-  were defined as NamedArguments where since they are required, the Name
-  component is not required.  This should not change operations except that
-  when we were mocking the methods, it returns sees the parameter as name=value
-  rather than value. See issue #833
+* Added `libxml2` operating system package as a dependency. It provides xmllint,
+  which is used for testing.
+
+* Fixed issue where `MOFCompiler.compile_str()` could not compile MOF that was
```
Comment (KSchopmeyer) [inline docs/changes.rst]: I think that is compile_string. Actually we should chane name in mock to _string to match.

Comment (KSchopmeyer) [reply docs/changes.rst]: I think it is already _string

Comment (andy-maier) [reply docs/changes.rst]: DONE

**Pull request merged**